apache/superset
# fix: Ch31968query context

### SUMMARY
The were some recent changes that are causing an error when reading a query context that has a dataset id tied to a dataset that does not exist. In the case where we found this error, the invalid dataset id was caused by exporting query_context which was then exporting the dataset id instead of a uuid or something that could be transferred from one superset instance to another. So this PR removes the query_context from export and import and also does not throw an error if a chart tries to read the query_context and hits a dataset not found error. 

In the near future, we should think about cleaning up any existing query contexts in a migration or script that point to a dataset that does not exist. 

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
n/a

### TESTING INSTRUCTIONS
export a dashboard with charts that have query_context and import them. When loading the dashboard in the new instance, there should be no errors. 

### ADDITIONAL INFORMATION
